extensions/blocks/simple-payments/edit.js

```diff
@@ -514,19 +533,21 @@ class SimplePaymentsEdit extends Component {
 							onChange={ this.handleMultipleChange }
 						/>
 					</div>
-
-					<TextControl
-						aria-describedby={ `${ instanceId }-email-${ fieldEmailError ? 'error' : 'help' }` }
-						className={ classNames( 'simple-payments__field', 'simple-payments__field-email', {
-							'simple-payments__field-has-error': fieldEmailError,
-						} ) }
-						label={ __( 'Email', 'jetpack' ) }
-						onChange={ this.handleEmailChange }
-						placeholder={ __( 'Email', 'jetpack' ) }
-						required
-						type="email"
-						value={ email }
-					/>
+					{ /* temporary wrapper until Gutenberg paste handler is bypassed for inputs with type="email" */ }
+					<div ref={ this.emailInput }>
```

Comment:
Need a wrapper div as can't add the ref to `TextControl` as it isn't a class component.